library/src/main/java/org/xmtp/android/library/ConversationV1.kt

```diff
@@ -195,6 +195,16 @@ data class ConversationV1(
     }
 
     suspend fun send(prepared: PreparedMessage): String {
+        if (client.v3Client != null) {
+            try {
+                val dm = client.conversations.findOrCreateDm(peerAddress)
+                prepared.encodedContent?.let {
+                    dm.send(it)
+                }
+            } catch (e: Exception) {
+                // Do nothing if this errors
+            }
+        }
```

Comment:
Catch all errors and do nothing if the message can't be sent or dm created. Do our best to create V3 equivalent dms but don't error if it doesn't happen.